### PR TITLE
Add `installPods` to legacy init

### PR DESCRIPTION
**Pull request by lucasbento**
Summary:
---------

This PR is related to #368 and adds `installPods` to the legacy init.

Test Plan:
----------

Follow https://github.com/react-native-community/cli/blob/master/CONTRIBUTING.md#testing-init-command.

### CODE REVIEW

File: packages/cli/src/tools/installPods.js [hunk 1]
```diff
@@ -26,7 +26,9 @@ async function installPods({
     try {
       await commandExists('pod');
     } catch (e) {
-      loader.stop();
+      if (loader) {
+        loader.stop();
```
Comment (grabbou) [inline packages/cli/src/tools/installPods.js]: A bit ugly, but I guess there's nothing we can do about it!

**Comment by grabbou** [comment]
Looks good to me!

File: packages/cli/src/tools/installPods.js (hunk 1 above)
Comment (lucasbento) [reply packages/cli/src/tools/installPods.js]: Thought the same.

Comment (thymikee) [reply packages/cli/src/tools/installPods.js]: you could add "onInstallFailure" and "onInstallSuccess" callbacks and delegate to them to make it cleaner, but I don't care about it here. It's not _that_ ugly provided it's a small helper used in 2 places, not a public API surface :)

**Pull request merged**